azure-pipelines/build.yml

```diff
@@ -0,0 +1,95 @@
+name: $(Date:yyyyMMdd).$(Rev:r)
+variables:
+- name: Codeql.Enabled
+  value: true
+- name: PublishNuGetOrg
+  value: false
+- name: TeamName
+  value: vscx-tools-authoring
+- name: TimestampPackage
+  value: true
+trigger:
+  branches:
+    include:
+    - refs/heads/main
+resources:
+  repositories:
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: 1ESPT-Windows2022
+      os: windows
+
+    stages:
+    - stage: Build
+      jobs:
+      - job: Build
+        displayName: 'Build'
+        timeoutInMinutes: 120
+        templateContext:
+          outputs:
+          - output: nuget
+            displayName: 'NuGet push to VS feed'
+            condition: and(succeeded(), eq(variables['Build.OfficialRelease'], 'true'))
+            packageParentPath: '$(Build.ArtifactStagingDirectory)\Packages'
+            packagesToPush: $(Build.ArtifactStagingDirectory)\Packages\*.nupkg;!$(Build.ArtifactStagingDirectory)\Packages\*.symbols.nupkg
+            publishVstsFeed: 97a41293-2972-4f48-8c0e-05493ae82010
+          - output: nuget
+            displayName: 'NuGet push to NuGet.org'
+            condition: and(and(succeeded(), eq(variables['Build.OfficialRelease'], 'true')), eq(variables['TimestampPackage'], 'false'))
+            packageParentPath: '$(Build.ArtifactStagingDirectory)\Packages'
+            packagesToPush: $(Build.ArtifactStagingDirectory)\Packages\*.nupkg;!$(Build.ArtifactStagingDirectory)\Packages\*.symbols.nupkg
+            nuGetFeedType: external
+            externalEndpoint: XamlBehaviors-NuGet.org
```

Comment:
>XamlBehaviors-NuGet.org [](http://example.com/codeflow?start=30&length=23)

what is this feed?

Reply:
The azure pipeline yml file is currently just a "starting point" that I'll undoubtedly have to edit as I work on getting the pipeline to work.